mne/time_frequency/tests/test_stockwell.py

```diff
@@ -64,26 +66,36 @@ def test_stockwell_core():
 
     width = 1.0
     start_f, stop_f = 0, len(pulse)
-    st_precomputed = _precompute_st_windows(1000, start_f, stop_f, sfreq, width)
-    y = _st(pulse, *st_precomputed)
+    W = _precompute_st_windows(n_samp, start_f, stop_f, sfreq, width)
+    y = _st(pulse, start_f, W)
     # invert stockwell
     y_inv = fftpack.ifft(np.sum(y, axis=1)).real
-
     assert_array_almost_equal(pulse, y_inv)
 
 
+@slow_test
 def test_stockwell_api():
-    """test stockwell functions"""
+    """Test stockwell functions"""
     epochs = Epochs(raw, events,  # XXX pick 2 has epochs of zeros.
                     event_id, tmin, tmax, picks=[0, 1, 3], baseline=(None, 0))
     for fmin, fmax in [(None, 50), (5, 50), (5, None)]:
         power, itc = tfr_stockwell(epochs, fmin=fmin, fmax=fmax,
                                    return_itc=True)
+        power_par, itc_par = tfr_stockwell(epochs, fmin=fmin, fmax=fmax,
+                                           return_itc=True, n_jobs=2)
```

Comment:
if there is no branching anymore you shouldn't test anymore with n_jobs > 1

can you go over the tests to check this? it should make our tests faster too :)